dist/manifest.json

```diff
@@ -5,7 +5,6 @@
   "version": "1.0.0",
   "minimum_chrome_version": "93",
   "action": {
-    "default_icon": "logo.png",
```

Comment:
@0xmad Could you please reset this change since I will push the image "logo.png"? Thanks